app/base/utils.py

```diff
@@ -1,111 +1,39 @@
-import json
 import os
 import uuid
+import numpy as np
 from datetime import datetime
-from PIL import Image
-from flask_login import current_user
-from werkzeug.utils import secure_filename
-from app import db, s3
+from app import redis_client
 from config import S3_BUCKET_CONFIG
-from app.base.models import Property
 
 bucket = S3_BUCKET_CONFIG["S3_BUCKET"]
-s3_prop_image_dir = S3_BUCKET_CONFIG["PROP_ASSETS"]
-s3_temp_dir = S3_BUCKET_CONFIG["TEMP_DIR"]
-s3_user_image_dir = S3_BUCKET_CONFIG["USER_ASSETS"]
 
 
-def save_images_to_temp_folder(image_files):
+def generate_dir_name(username):
     """
-    Creates a temporary folder for storing image objects and returns
-    name of the temporary folder and the list of the filenames that were saved to S3 in the
-    same folder.
+    This function generates a random string to be used as a directory name where images will be saved on S3
     """
-    from app.tasks import image_uploader
-    dir_name = str(uuid.uuid4())[:8] + "/"
-
-    image_list = []
-    for image_file in image_files:
-        random_str = str(uuid.uuid4())[:13].replace("-", "")
-        checked_filename = secure_filename(image_file.filename)
-        _, file_ext = os.path.splitext(checked_filename)  # get file extension
-        clean_filename = f"{random_str}{file_ext}"
-        image_list.append(clean_filename)
-        image_uploader(image_file, s3_temp_dir + dir_name + clean_filename)
-
-    return dir_name, image_list
-
-
-def generate_img_dir_name():
-    """
-    This function generates a random string which is concatenated together with the current date and time, then
-    returns it. This will be used by the property_image_handler() and update_property_images() function as name for
-    the directory where the image files should be saved.
-    """
-    suffix = str(uuid.uuid4())[:15].replace("-", "")
+    suffix = uuid.uuid4().__str__()[:5].replace("-", "")
```

Comment:
a uuid string is of this format
`263f7b9a-fac1-4bad-aee9-3442bad3c042`
and then  when you slice it to get the first 5 characters only you get
`263f7`

do  you think there is anything you need to replace `replace("-", "")` in  this pattern?